build/configs/artik053/README.md

```diff
@@ -91,41 +91,24 @@ openocd -f artik053.cfg -c ' \
 
 ## ROMFS
 
-Before executing below steps, execute [generic steps](../../../tools/fs/README_ROMFS.md), step 1 and step 2.
+Execute [generic steps](../../../tools/fs/README_ROMFS.md), step 1 and step 2.
+If you want to change the size of ROMFS partition, execute below steps.
 
-3. Modify partition configs.  
-    Below steps creates ROMFS partition with size 400KB at next of user partition.  
-    1. Split user partition size from (1400) to (1000, 400) in ARTIK053_FLASH_PART_LIST
+1. Modify partition configs.
```

Comment:
Don't modify this to 1st.  
1 and 2 are in general steps so that this is a 3rd step.

Reply:
Ok. I need to modify it.

Reply:
Ok. I need to modify it.